pkg/db/migrations/8_config_component_relationships.sql

```diff
@@ -0,0 +1,21 @@
+-- +goose Up
+-- +goose StatementBegin
+
+CREATE TABLE config_component_relationships(
+    component_id UUID NOT NULL,
+    config_id UUID NOT NULL,
+    created_at timestamp NOT NULL DEFAULT now(),
+    updated_at timestamp NOT NULL DEFAULT now(),
+    deleted_at TIMESTAMP DEFAULT NULL,
+    FOREIGN KEY(component_id) REFERENCES components(id), 
+    FOREIGN KEY (config_id) REFERENCES config_items(id),
```

Comment:
We can't add this foreign key here, as the config_items may not have been created yet - it probably makes more sense to add the foreign key in incident commander